bin/release.sh

```diff
@@ -26,6 +26,20 @@ cd /app/bugsnag-js
 npm ci
 npm run bootstrap -- --ci
 
+# increment package version numbers
+if [ ! RETRY_PUBLISH ]; then
+  case $VERSION in
+    "prerelease" | "prepatch" | "preminor" | "premajor")
+      npx lerna version "$VERSION" --dist-tag next --no-push
+      ;;
+
+    *)
+      npx lerna version "$VERSION" --no-push
```

Comment:
this is to prevent git tags being pushed until after we've built as per the steps in the ticket. that being said - we could allow tagging during the version and remove the later step to push the tags?

Reply:
Ah yeah — I figured since we've already built on CI it would always work but I guess there's no harm in being paranoid in case something changes between CI running and this script running